examples/jsm/nodes/display/ViewportNode.js

```diff
@@ -0,0 +1,106 @@
+import Node from '../core/Node.js';
+import { uniform, div, vec2, invert } from '../shadernode/ShaderNodeBaseElements.js';
+import { Vector2 } from 'three';
+import { NodeUpdateType } from '../core/constants.js';
+
+let resolution;
```

Comment:
Maybe we should move this variable to instances? Because it is possible that a user would want to render for different sizes.

Reply:
I thought about it, but it shouldn't happen since `.update()` will force the update in GPU level every time that it is called.